src/res/instruments/djembe.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 19 27"><defs><style>.cls-1{fill:#242323;}</style></defs><path class="cls-1" d="M13.55,15.19h0l2.33-8.7-.14-.63a.58.58,0,0,0,0-.19c0-1.27-2.8-2.3-6.26-2.3-3,0-5.49.77-6.1,1.81h0v0a.92.92,0,0,0-.14.47s0,.08,0,.12l-.15.7L6,15.1c0,.05.06.19.48.5l-1.34,6a6,6,0,0,0,4.42,2,8.87,8.87,0,0,0,4.83-1.5L13,15.71Zm-.5-6.36a20.36,20.36,0,0,1-1.14,5.34c-.1.29-.57.17-.46-.12a19.86,19.86,0,0,0,1.12-5.22C12.59,8.52,13.07,8.52,13.05,8.83ZM9.59,3.69c2.9,0,5.25.56,5.25,1.26S12.49,6.21,9.59,6.21,4.34,5.64,4.34,5,6.69,3.69,9.59,3.69Zm-5,3.22a11.73,11.73,0,0,0,2,.47,22.69,22.69,0,0,0,8-.47c.31-.06.43.4.13.47-.76.15-1.52.29-2.28.4A22.9,22.9,0,0,1,8.26,8a13.76,13.76,0,0,1-3.82-.63C4.14,7.28,4.27,6.81,4.56,6.91Zm5.3,7.6a.24.24,0,0,1-.48,0,35.36,35.36,0,0,1-.21-5.44c0-.31.49-.31.48,0A35.36,35.36,0,0,0,9.86,14.51Zm-3.09-.24a37.23,37.23,0,0,1-.92-5.44c0-.31.46-.31.49,0a35.46,35.46,0,0,0,.89,5.31C7.31,14.44,6.84,14.57,6.77,14.27Zm5.47,1.79a16.17,16.17,0,0,1-2.59.29,7.86,7.86,0,0,1-2.37-.23.24.24,0,0,1,.13-.46,7.35,7.35,0,0,0,2.24.21,14.48,14.48,0,0,0,2.46-.28C12.41,15.54,12.54,16,12.24,16.06Z"/></svg>
```

Comment:
Do we have a style guide for XML-ish files?  "Pretty" XML, plus a copyright header?
```
<!-- Copyright YYYY Jamulus Development team
        author: Riley-I (https://github.com/Riley-I) -->
```

Reply:
Ah yes, please keep this in mind if you add svg files.